zap/zap.gradle.kts

```diff
@@ -89,7 +89,7 @@ dependencies {
     api("org.jfree:jfreechart:1.5.3")
     api("org.jgrapht:jgrapht-core:0.9.0")
     api("org.swinglabs.swingx:swingx-all:1.6.5-1")
-    api("org.xerial:sqlite-jdbc:3.36.0.3")
+    api("org.xerial:sqlite-jdbc:3.39.3.0")
```

Comment:
Not really about this PR but I added a task to #3113 to make this an implementation dependency. Core only needed it for the spider and since the spider is now in an add-on there's no reason to keep bundling in core for much longer, so the sooner we hide it the better. Guess we should add it to `commonlib`, `spider` and `ascanrulesBeta` use it.